### DISCUSSION TITLE
How can I confirm that element is selected using appium flutter driver?

### DISCUSSION THREAD

**Discussion by Harsha-Gelato** [appium/appium#17331] [General]
Hi All,

I am currently automating one testCase in which I have to verify "**English**" as the language is selected.
I am using [appium flutter driver](https://github.com/appium-userland/appium-flutter-driver) but here I did not able to find any function which will help me to verify such testCase.
![Screenshot_20220729_092006](https://user-images.githubusercontent.com/103107117/183422123-4d3fc8ca-2b0f-4fd5-bcee-c75e4a21a5bb.png)

Can someone please suggest me the best option for the same. Thanks :)

**Comment by KazuCocoa** [comment]
appium flutter driver sends flutter_driver commands in https://api.flutter.dev/flutter/flutter_driver/flutter_driver-library.html, so the driver can only do something provided by the flutter_driver package in FLUTTER context.
Could you find something that could help to detect the element value in `getRenderTree` command?
